src/platform-implementation-js/dom-driver/gmail/views/gmail-route-view/gmail-route-view.ts

```diff
@@ -392,42 +420,8 @@ class GmailRouteView {
   }
 
   async _startMonitoringPreviewPaneForThread(
```

Comment:
This method isn't called outside this class, and this class isn't part of our public interface.

```suggestion
  async #startMonitoringPreviewPaneForThread(
```

Reply:
changed